File: src/main/java/com/shopping/shopping_site_backend/infra/constant/enumconstant/PaymentType.java
package com.shopping.shopping_site_backend.infra.constant.enumconstant;

public enum PaymentType {

    CREDIT_CARD("Credit Card"),  // 信用卡
    BANK_TRANSFER("Bank Transfer"),  // 银行转账
    CASH_ON_DELIVERY("Cash on Delivery") // 货到付款
    ;
    private String description;
    PaymentType(String description) {
        this.description = description;
    }
}

File: src/main/java/com/shopping/shopping_site_backend/infra/constant/enum/PaymentType.kt
package com.shopping.shopping_site_backend.infra.constant.enum

enum class PaymentType(val type: String) {
    CREDIT_CARD("Credit Card"),  // 信用卡
    BANK_TRANSFER("Bank Transfer"),  // 银行转账
    CASH_ON_DELIVERY("Cash on Delivery") // 货到付款
}
